packages/frontend/src/components/bridge/BridgeDetails.tsx

```diff
@@ -26,8 +26,7 @@ export const BridgeDetails = ({ bridge }: BridgeDetailsProps) => {
             <CommissioningInfo bridge={bridge} />
           </Grid>
         </Grid>
-
-        <Stack direction="row" spacing={1}>
+        <Stack direction="row" spacing={1} sx={{ flexWrap: "wrap" }} useFlexGap>
```

Comment:
`Stack` has a property for `flexWrap`. No need to use `sx`.

Reply:
Nice catch, a changed it and removed the sx.